src/components/Pagination/Pagination.st.css

```diff
@@ -0,0 +1,198 @@
+
+:import {
+  -st-from: "wix-ui-core/index.st.css";
+  -st-named: Pagination
+}
+
+:import {
+  -st-from: "../../tpaStyles/styleableExt.js";
+  -st-named: wixApply;
+}
+
+:vars {
+  defaultTextFont: "{theme: 'Body-M', size: '16px'}";
+  defaultTextFontColor: color-5;
+  defaultSelectedTextFontColor: color-8;
+  defaultDisabledTextFontColor: color-3;
+}
+
+:vars {
+  TextFont: --overridable;
+  TextFontColor: --overridable;
+  SelectedTextFontColor: --overridable;
+  DisabledTextFontColor: --overridable;
+}
+
+.root {
+  -st-extends: Pagination;
+  -st-states: rtl;
+  
+  align-items: stretch;
+  box-sizing: border-box;
+  display: inline-flex;
+  font:  wixApply(fallback, wixApply(font, value(TextFont)), wixApply(font, value(defaultTextFont)));
+}
+
+.pageButtonRules {
+  min-width: 1em;
+  min-height: 1em;
+  line-height: 1em;
+  justify-content: center;
+  align-items: center;
+  padding: 0.5em 0.5em;
+  margin: 0.25em;
+}
+
+.root::pageButton {
+  -st-mixin: pageButtonRules;
+  position: relative;
+  color: wixApply(color, wixApply(fallback, value(TextFontColor), value(defaultTextFontColor)));
+}
+
+.root::currentPage {
+  -st-mixin: pageButtonRules;
+  color: wixApply(color, wixApply(fallback, value(SelectedTextFontColor), value(defaultSelectedTextFontColor)));
+  padding: 0.5em;
+}
+
+.root:not(:disabled)::pageButton:hover::before {
+  opacity: 0.2;
+  content: "";
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+}
+
+.root:not(:disabled)::pageButton:hover {
+  opacity: 0.7;
```

Comment:
Designer asked to add a `.2s` transition on hover

Reply:
done =)